payments-core/src/test/java/com/stripe/android/PaymentMethodEndToEndTest.kt

```diff
@@ -376,4 +376,14 @@ internal class PaymentMethodEndToEndTest {
 
         assertThat(paymentMethod?.type).isEqualTo(PaymentMethod.Type.Klarna)
     }
+
+    @Test
+    fun `createPaymentMethod with Affirm should create expected object`() {
+        val paymentMethod = Stripe(context, ApiKeyFixtures.AFFIRM_PUBLISHABLE_KEY)
```

Comment:
Is shipping address required? If so we should make a unit test similar to the klarna one right above and test for it.

Reply:
I tried to curl the api to create an affirm payment method. It did succeed without address. So looks like the address is not technically required.